deploy/examples/cluster.yaml

```diff
@@ -281,6 +281,31 @@ spec:
     # No values or 0 means that the operator will wait until the placement groups are healthy before unblocking the next drain.
     pgHealthCheckTimeout: 0
 
+  # csiDriverOptions defines defines CSI Driver settings applied per cluster.
+  csiDriverOptions:
+    # common settings for all CSI drivers.
+    common:
```

Comment:
I strongly plus-one this. Any configs that are shared by all CSI drivers should not exist on CephClusters. With 'common' settings, different CephClusters could compete to be the source of truth for the shared CSI settings. Competition is easily solved by keeping common settings separate, in a singleton. Currently, the CSI common config is the rook-ceph-operator-config configmap, but it could be a separate CRD in the future, like COSI has done.